doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -259,26 +260,16 @@ to the specified type. For example, the following call:
 ccall((:foo, "libfoo"), Cvoid, (Int32, Float64), x, y)
 ```
 
-will behave as if the following were written:
+will behave as if the following were written[^3]:
 
 ```julia
 ccall((:foo, "libfoo"), Cvoid, (Int32, Float64),
-      Base.unsafe_convert(Int32, Base.cconvert(Int32, x)),
-      Base.unsafe_convert(Float64, Base.cconvert(Float64, y)))
+      Base.convert(Int32, x), Base.convert(Float64, y))
```

Comment:
Why change this and remove the text below? This is not how it works and gives the wrong impression.

Reply:
The text below is preserved as a footnote.  I wrote it this way because I felt that, for most users, `cconvert` followed by `unsafe_convert` was more detail than they would want to know, and that this gets the point across (i.e., that the passed values are converted to the appropriate type).  That said, I'm willing to change it back if you don't think the footnote is enough or appropriate.

Reply:
I mean, this is the docs for how to `ccall` so why should I have to scroll to a footnote in order to see how `ccall` works?

Reply:
>  for most users, cconvert followed by unsafe_convert was more detail than they would want to know, and that this gets the point across (i.e., that the passed values are converted to the appropriate type).

For any user, they should either not care about anything about conversion for `ccall` arguments at all, or if they do, they must not related it to `convert`. This change gets the wrong point across. I've recently seen people still thinking `convert` is the method to use for this and it's just wrong.

Bottom line is that I don't really mind if this is said in the footnote or not, but you must not mention `convert`.